### DISCUSSION TITLE
wiki markdown not rendered

### DISCUSSION THREAD

**Discussion by Papa-26** [community/community#49409] [New to GitHub]
I have added a markdown file to a wiki that rendered properly  in Intellij, and in the non-wiki part of the repository, but in the wiki it doesn't render:
![Screenshot from 2023-03-07 06-15-30](https://user-images.githubusercontent.com/83922598/223410166-95a5744b-1ee1-4ee8-b3f7-81b61c13e602.png)
The browser tag designates it as 'raw,,,usercontent'
What did I do wrong?

**Comment by GregoryNavasarkian** [comment]
I would recommend adding the markdown manually on GitHub. Other than that it could be a bug or formatting issue.
